templates/vhost/_ssl.erb

```diff
@@ -4,22 +4,22 @@
   SSLEngine on
   SSLCertificateFile      "<%= @ssl_cert %>"
   SSLCertificateKeyFile   "<%= @ssl_key %>"
-  <%- if @ssl_chain -%>
+  <%- if @ssl_chain && @ssl_chain != '' -%>
```

Comment:
I will defer to you, I am new to Ruby and was just copying what was already done in the template for SSLCACertificatePath.